hubblestack/utils/raetevent.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+'''
+Manage events
```

Comment:
Please check if this file is relevant. We don't use events

Reply:
This file is used by event utility which is needed in roots fileserver.